### PR TITLE
Removed Course Info Placeholders

### PR DESCRIPTION
**Pull request by clintonb**
We won't have this information for launch, so the placeholders need to go away.

@dsjen @rocha

### CODE REVIEW

File: analytics_dashboard/courses/views.py [hunk 1]
```diff
@@ -39,14 +39,8 @@ def get_default_data(self):
         """
         Returns default data for the pages (context and javascript data).
         """
-        # TODO: Use real course data
         user = self.request.user
         context = {
-            'user_name': user.get_full_name(),
```
Comment (dsjen) [inline analytics_dashboard/courses/views.py:45]: This user is also referenced in js_data (below).

Comment (clintonb) [inline analytics_dashboard/courses/views.py:45]: Okay. The `user` variable still remains. Only the context is being updated.

**Comment by clintonb** [comment]
@dsjen Any other feedback?

File: analytics_dashboard/courses/views.py (hunk 1 above)
Comment (dsjen) [inline analytics_dashboard/courses/views.py:45]: I think the user information in the js_data should go away too; I don't think there will be any repercussions for removing it there too.

Comment (clintonb) [inline analytics_dashboard/courses/views.py:45]: It is used for tracking.

Comment (dsjen) [inline analytics_dashboard/courses/views.py:45]: What user information is valid?  I think it makes sense to remove the invalid information from tracking as well then.

Comment (clintonb) [inline analytics_dashboard/courses/views.py:45]: Didn't you write the page tracking? All of the information below is being sent to SegmentIO.

Comment (dsjen) [inline analytics_dashboard/courses/views.py:45]: It seems to me that this PR is about removing the invalid user information as it won't be available for launch.  My suggestion is that it be removed from segment.io too.  The original assumption for including it to segment.io was that it was (or would be) valid.  If there is some identifying information that we can send, we should still do so.

**Comment by dsjen** [comment]
Confusion cleared up!  Looks good to me.

**Comment by clintonb** [comment]
:smile: Thanks.

**Pull request merged**